portal-api/api-tests/v1/bonds/bonds.api-test.js

```diff
@@ -85,9 +89,9 @@ describe('/v1/deals/:id/bond', () => {
   };
 
   beforeAll(async () => {
-    const testUsers = await testUserCache.initialise(app);
+    testUsers = await testUserCache.initialise(app);
 
-    noRoles = testUsers().withoutAnyRoles().one();
+    noRoles = testUsers().withoutAnyRoles().withBankName('Barclays Bank').one();
     aBarclaysMaker = testUsers().withRole('maker').withBankName('Barclays Bank').one();
```

Comment:
constants? :)

Reply:
There are ~700 results for the string barclays across DTFS and it would cause a lot of merge conflicts if we tried to change them all in this PR so I've raised a tech debt ticket to change this, [DTFS2-6685](https://ukef-dtfs.atlassian.net/browse/DTFS2-6685)